src/components/molecules/TablesUserList/TablesUserList.tsx

```diff
@@ -58,6 +61,7 @@ const TablesUserList: React.FunctionComponent<PropsType> = ({
   const [showJoinMessage, setShowJoinMessage] = useState(false);
   const [joiningTable, setJoiningTable] = useState("");
   const [videoRoom, setVideoRoom] = useState("");
+  const startTable = true;
```

Comment:
Condition to render "Start a Table" button

Reply:
Pending further clarity from @sparkletown/product presumably?

Reply:
condition described here https://github.com/sparkletown/internal-sparkle-issues/issues/852#issuecomment-874640768

Reply:
fixed here b3efcb0a743aa87f509a55971b8998bb30e2dd41